spi/common/core-spi/src/main/java/org/eclipse/edc/spi/command/CommandHandler.java

```diff
@@ -31,6 +33,6 @@
     /**
      * Processes the command.
      */
-    void handle(T command);
+    Result<Void> handle(T command);
```

Comment:
Now that the `CommandHandler` returns a `Result`, the `CommandRunner` can return it's result (as specified in the issue #2340 )